city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,129 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/board-meeting-materials"]
+
+    def parse(self, response):
+        self.link_date_map = defaultdict(list)
+        link_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]/h3'
+        )
+        for title in link_list:
+            full_str = title.xpath(".//text()").extract()[0]
+            date = full_str.split("-")[0].split("(")[0]
```

Comment:
They may change the characters used after this, so checking for a date match with regular expressions might be safer:

```python
date_match = re.search(r"[a-z]{3,9} \d{1,2},? \d{4}", full_str, flags=re.I)
```
That would check for a case-insensitive match that still matches if the comma isn't there, and you can just remove the comma in `strptime` to be safe